### PR TITLE
[smartswitch] Add support of a new 't1-smartswitch' topology to the sample config generator.

### PR DESCRIPTION
**Pull request by oleksandrivantsiv**
#### Why I did it
Add support of a new 't1-smartswitch' topology to the sample config generator. The topology passed to sonic-cfggen utility as a parameter to generate sample configuration for Smart Switch:
```
sonic-cfggen  -k <SKU> --preset t1-smartswitch ...
```

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Extend sample config generator to support new topology and read Smart Switch specific data from hwsku.json.

#### How to verify it
Run unit tests. The changes are covered with the new unit tests. 

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211
- [ ] 202305

#### Tested branch (Please provide the tested image version)

- [ ] 
- [ ]

### CODE REVIEW

File: src/sonic-config-engine/config_samples.py [hunk 1]
```diff
@@ -75,6 +75,39 @@ def generate_t1_sample_config(data):
...
 
+def generate_t1_smartswitch_sample_config(data):
+    data = generate_t1_sample_config(data)
+    data['DEVICE_METADATA']['localhost']['subtype'] = 'SmartSwitch'
+
+    bridge_name = 'bridge_midplane'
+    data['MID_PLANE_BRIDGE'] = {
+        'GLOBAL': {
+            'bridge': bridge_name,
+            'address': '169.254.200.254/24'
```
Comment (Yakiv-Huryk) [inline src/sonic-config-engine/config_samples.py]: does it make sense to have the 169.254.200.254 defined in a single place?

File: src/sonic-config-engine/smartswitch_config.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        import mock_tables.dbconnector
+        mock_tables.dbconnector.load_namespace_config()
+
+except KeyError:
+    pass
+
+DPUS_TABLE  = 'DPUS'
+
+
+def get_smartswitch_cofig(hwsku=None, platform=None):
```
Comment (dgsudharsan) [inline src/sonic-config-engine/smartswitch_config.py]: Please correct typo get_smartswitch_config*

**Review by dgsudharsan — CHANGES_REQUESTED** [review]

File: src/sonic-config-engine/config_samples.py (hunk 1 above)
Comment (oleksandrivantsiv) [reply src/sonic-config-engine/config_samples.py]: done

File: src/sonic-config-engine/smartswitch_config.py (hunk 2 above)
Comment (oleksandrivantsiv) [reply src/sonic-config-engine/smartswitch_config.py]: done

**Review by dgsudharsan — APPROVED** [review]

**Pull request closed without merging**